Add Ruby support

kassio opened the pull request:
https://user-images.githubusercontent.com/120483/134683022-171d7271-0465-4853-b1ab-a30c79c1a161.mov

SmiteshP commented:
Thanks for the PR! 😄 
Could you share some code snippets or screenshots for these queries, so that I have a reference in future.
In particular, I want to see how the last two queries work because they are using scope-root-2 (maybe that's a mistake 🤔) but have a single capture inside them.

kassio commented:
@SmiteshP, I think you were right, I copied that from another query, thanks for pointing that up! 😄 

I updated the query and added a video on the PR description, can you take another look, please?

SmiteshP reviewed (approved):
LGTM :)
